packages/cli/src/commands/generate/function/__tests__/fixtures/jsFunc.js

```diff
@@ -1,18 +1,34 @@
+import { logger } from 'src/lib/logger'
+
 /**
  * The handler function is your code that processes http request events.
  * You can use return and throw to send a response or error, respectively.
  *
+ * Important: When deployed, a custom serverless function is an open API endpoint and
+ * is your responsibility to secure appropriately.
+ *
+ * @see {@link https://redwoodjs.com/docs/serverless-functions#security-considerations|Serverless Function Considerations}
+ * in the RedwoodJS documentation for more information.
+ *
  * @typedef { import('aws-lambda').APIGatewayEvent } APIGatewayEvent
  * @typedef { import('aws-lambda').Context } Context
  * @param { APIGatewayEvent } event - an object which contains information from the invoker.
  * @param { Context } context - contains information about the invocation,
  * function, and execution environment.
  */
 export const handler = async (event, context) => {
-  return {
-    statusCode: 200,
-    body: JSON.stringify({
-      data: 'javascriptFunction function',
-    }),
+  try {
+    return {
+      statusCode: 200,
+      body: JSON.stringify({
+        data: 'javascriptFunction function',
+      }),
+    }
+  } catch (error) {
+    logger.error({ error }, error.message)
+
+    return {
```

Comment:
Want to show that a 401 unauthorized is appropriate in some cases.

When the `secureHandler` work is done in another PR that provides verification methods can leverage that here.

Reply:
The try/ catch part is a bit strange for me, since the return statement can't fail. Maybe a comment feel less strange until the secure handler work is done?

Reply:
> Maybe a comment feel less strange until the secure handler work is done?

Agreed. I'll remove until I can hook in `secureHandler`.

I was actually thinking `secureHandler` *could* have a verification method like `public` or `open` that always returns true so that that could be baked into the function template.

Then one just swaps out the verification method to SHA256 or whatnot.

Reply:
@peterp I couldn't think of a good way to comment the 401 case, so I simply removed the try/catch example. but I left in a small log statement as wanted to get logger into the template.

Once secureHandler is done I'll revisit this template.